src/pages/api/logout.ts

```diff
@@ -9,21 +9,21 @@ import {
 import { NextApiRequest, NextApiResponse } from "next";
 
 import { Handler } from "@src/@types/handler";
-import { DeleteUserCookieError } from "@src/errors";
 import { logger } from "@src/logger";
 import { withSentry } from "@src/middlewares/with-sentry";
 import getTracer from "@src/tracer";
-
-const tracer = getTracer();
+import { ERRORS_DATA, webErrorFactory } from "@src/web-errors";
 
 const handler: Handler = (_request, response): Promise<void> => {
-  return tracer.span("logout handler", async (span) => {
+  const webErrors = {
+    unexpectedError: ERRORS_DATA.UNEXPECTED_ERROR,
+  };
+
+  return getTracer().span("logout handler", async (span) => {
     await deleteServerSideCookie(response, "user-cookie").catch(() => {
       span.setDisclosedAttribute("user logged out", false);
 
-      throw new DeleteUserCookieError(
-        "Error deleting UserCookie for login out",
-      );
+      throw webErrorFactory(webErrors.unexpectedError);
```

Comment:
Should we be alerted when this error occurs? because in case of weberrors, we will wrap the error as something expected.

Reply:
Good point. I think we definitely want to be alerted.